### ISSUE TITLE
Is this for importing new google row to trello cards? If so could you please tell me how to setup?

### ISSUE THREAD

**Issue by DillonMercz** [chezmatt/riot-trello-connector#1]

**Comment by chezmatt** [comment]
Dillon! Not sure how you even found that code it's so old.
If you're not comfortable with APIs,
I recommend Zapier as the API specific to this script has changed since
then.

Otherwise setting up an instance to listen and handle the requests is
pretty straight forward. Google and Rock Creek are very good at
documenting. There's loads of google sheets packages on NPM for that. The
Trello API is really adaptive so you can push an imperfect object to your
sheet  and it will figure things out which is awesome.

Not sure if that helps.
__________________________
Matt Wilkinson
mlwilkinson@icloud.com

T: 650.308.9714
C:401.339.6507
650 Castro St #120-423
Mountain View, CA 94041 USA

**Comment by DillonMercz** [comment]
Yeah it was super old, but I am looking for a solution for a company I am
starting. We take customer requests through a custom form I made and it is
connected to the backend of a google form. The results end up in a
spreadsheet and from there I wanted it to go into trello.i am not good with
npm, But I am great with python. The code seemed pretty straightforward
where you give it the sheets api credentials and then it generates the
tokens for Trello and passes the info along. However like I said, I am not
good with javascript so I couldn't figure out how to run it.

Dylan Clayton
